SQL/syntax_test_sql.sql

```diff
@@ -28,3 +28,6 @@ multiline comment
 -- ^ source.sql comment.block.c
 */
 select
+
+
+<=>  -- <- keyword.operator.comparison.sql
```

Comment:
This is not a proper test line. Test lines should be specified in the lines *after* the line you want to test, so there should be a newline before `--`.

Reply:
Oh yeah, this is just testing the beginning token.  I'm pretty sure the following will be comprehensive though:

```sql
   <=>  
-- ^^^ keyword.operator.comparison.sql
```

Reply:
Sorry, I'm new to this. Didn't write a single test case in my life. Should I do another commit?

Reply:
Yes. Just change your test line above into the two lines that I wrote.